package.json

```diff
@@ -37,14 +37,15 @@
     "eslint-config-standard": "^10.2.1",
     "eslint-config-semistandard": "^11.0.0",
     "eslint-plugin-header": "^1.0.0",
-    "tap": "~10.7.2"
+    "tap": "~10.7.2",
+    "commander": "^2.11.0"
   },
   "browser": {
     "request": "browser-request"
   },
   "main": "./api.js",
   "scripts": {
-    "test": "eslint --ignore-path .eslintignore . && tap test/*.js"
+    "test": "eslint --ignore-path .eslintignore . && tap --timeout=900000 test/*.js"
```

Comment:
Each test sets a timeout, is this necessary?

Reply:
I was messing with this for a while. The `--timeout` flag here (which defaults to 30secs if not specified) seems to take priority over the timeouts set in the tests. Currently this gives a 15min timeout per test file.
An alternative would be to disable the timeout completely here (with `--no-timeout`) and set timeouts in all the other test files.